rhscl-mariadb-apb/apb.yml

```diff
@@ -0,0 +1,47 @@
+id: a8e652c1-11e0-4202-b6df-09517f75ef9a
+name: rhscl-mariadb-apb
+image: registry.access.redhat.com/rhscl/mariadb-101-rhel7
+#image: ansibleplaybookbundle/rhscl-mariadb-apb
+description: mariadb apb implementation
+bindable: true
+async: optional
+tags:
+  - database
+  - rhscl-mariadb
+metadata:
+  displayName: "RHSCL MariaDB"
+  longDescription: "Deploys a bindable mariadb instance"
+  imageUrl: "https://raw.githubusercontent.com/docker-library/docs/74e3b3d4d60389208732dbd2c95145868111d959/mariadb/logo.png"
+  documentationUrl: "https://mariadb.com/kb/en/mariadb/documentation/"
+parameters:
```

Comment:
The openshift catalog recently change the format for parameters. It looks like mariadb_version is correct, but the rest are wrong.

Use this format. You can exclude description where it's obvious. I use it as a hint below to explain behavior. title is what will be displayed next to the field in the UI. description underneath.

```
  - postgresql_database:
      title: PostgreSQL Database Name
      type: string
      default: admin
  - postgresql_password:
      title: PostgreSQL Password
      description: A random alphanumeric string if left blank
      type: string
```